### PR TITLE
Introduce create database with typeorm-extension apply to appModule

### PR DESCRIPTION
**Pull request by gabrielmatau79**
Introduce create database with typeorm-extension apply to appModule, has been tested on local and docker compose method.

Note: The file 'data-source.ts' is located in the 'src' folder. It is required by the TypeORM extension to create the database. It contains the necessary configuration for building the connection and database itself.

### CODE REVIEW

File: .prettierrc
Comment (genaris) [inline .prettierrc:1]: This is not related to this PR

File: src/data-source.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { DataSource, DataSourceOptions } from 'typeorm'
+import { SeederOptions } from 'typeorm-extension'
+
+export const dataSourceOptions: DataSourceOptions & SeederOptions = {
+  type: 'postgres',
+  host: process.env.POSTGRES_HOST ?? 'localhost',
+  port: parseInt(process.env.POSTGRES_PORT ?? '5432'),
+  database: 'trustregistry',
+  username: process.env.POSTGRES_USER ?? 'cloud-agent',
+  password: process.env.POSTGRES_PASSWORD ?? 'cloud-agent',
```
Comment (genaris) [inline src/data-source.ts:14]: I prefer to keep using 'trustregistry' as default user for dev. I don't want to link this project to cloud-agent

File: src/models/db/postgres.db.ts [hunk 2]
```diff
@@ -14,8 +14,8 @@ export const dbConfig: TypeOrmModuleOptions = {
   type: 'postgres',
   host: process.env.POSTGRES_HOST ?? 'localhost',
   port: parseInt(process.env.POSTGRES_PORT ?? '5432'),
-  username: process.env.POSTGRES_USER ?? 'trustregistry',
-  password: process.env.POSTGRES_PASSWORD ?? 'trustregistry',
+  username: process.env.POSTGRES_USER ?? 'cloud-agent',
```
Comment (genaris) [inline src/models/db/postgres.db.ts:17]: As mentioned previously, I think we can keep 'trustregistry' default user/pass for dev

**Review by genaris — COMMENTED** [review]
This PR supersedes #3, right? If that's the case please close the other one.

File: .prettierrc
Comment (gabrielmatau79) [reply .prettierrc:1]: I didn't have the configuration to remove the semicolom...

File: src/data-source.ts (hunk 1 above)
Comment (gabrielmatau79) [reply src/data-source.ts:14]: rollback to user and password

File: src/models/db/postgres.db.ts (hunk 2 above)
Comment (gabrielmatau79) [reply src/models/db/postgres.db.ts:17]: rollback to user and password

**Review by genaris — APPROVED** [review]
